ui/components/fixture-physical.vue

```diff
@@ -117,17 +135,21 @@
   column-gap: 3rem;
 
   & section {
-    break-inside: avoid;
```

Comment:
Why did you remove this? It prevents column breaks between a heading and the following content.